quirks/password-rules.json

```diff
@@ -5,6 +5,12 @@
     "1800flowers.com": {
         "password-rules": "minlength: 6; required: lower, upper; required: digit;"
     },
+    "access.service.gov.uk": {
+        "password-rules": "required: lower; required: upper; required: digit; required: special; minlength: 10;"
+    },
+    "admiral.com": {
+        "password-rules": "required: digit; required: [- !&quot;#$&'()*+,.:;<=>?@[^_`{|}~]]; allowed: upper,lower; minlength: 8;"
```

Comment:
`&quot;`?

Reply:
Whoops I copied the HTML-escaped rules by accident. Will do a pass through the added sites and fix them before submitting the PR 👍